preface.asciidoc

```diff
@@ -253,3 +253,4 @@ Following is an alphabetically sorted list of notable GitHub contributors, inclu
 * Yeramin Santana (ysfdev)
 * Zhen Wang (zmxv)
 * ztz (zt2)
+* Pet3rpan (pet3r-pan)
```

Comment:
If you want, you can locate your name by orthographic order. For this, please publish a new PR with that change.